db/migrate/20220318023014_create_members.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMembers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :members do |t|
+      t.string :name, null: false
+      t.string :facebookUrl
+      t.string :instagramUrl
+      t.string :linkedinUrl
```

Comment:
One detail I forget to mention is that there is a convention of using `lower_snake_case` in compound database field names.

Reply:
I found [this gist ](https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e)that might help you.

Reply:
in this case it would be better to leave the data as "facebook_url" for example?